wikidp/controller.py

```diff
@@ -82,12 +82,15 @@ def contribute_selected_page():
     preview_item = DF.item_detail_parse(request.form['qid'])
     return render_template('contribute.html', selected=preview_item, options=options)
 
-@APP.route("/<qid>")
-def selected_item(qid):
-    """If the item ID is already known, the user can enter in the url, not yet functioning"""
-    qid = qid.strip()
-    properties()
-    return preview_selected_page()
+@APP.route("/q<id>")
```

Comment:
Is this really case sensitive?

Reply:
To recap from our dev meeting on this: Flask by default handles routing case sensitively 
Similar issue in stackoverflow: (I like the top answer's 404 approach as something to consider in the future)
https://stackoverflow.com/questions/28801707/case-insensitive-routing-in-flask